media/js/mozorg/contribute/friends.js

```diff
@@ -1,4 +1,13 @@
-$(function() {
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function($, Mozilla) {
+    'use strict';
+
+    var $ffShowSignupForm = $('#ff-show-signup-form');
+    var $ffSignupArea = $('#ff-signup');
+    //var $ffSignupButton = $('#footer_email_submit');
 
     function supportsSVG() {
```

Comment:
I recently wrote a reusable component that does this here https://github.com/mozilla/bedrock/blob/master/media/js/base/mozilla-svg-image-fallback.js